lib/entity_manager.js

```diff
@@ -37,6 +40,8 @@ EntityManager.prototype.create = function() {
   var entity = this._entityInst[id]
   this._entityFlag[id] = ENTITY_ALIVE
 
+  entity._cacheIndex = this._entityCache[0].push(entity) - 1
```

Comment:
Do not add anything to the entity.
Instead create an `Uint32Array` to store indexes like `_entityMask`.